static/js/compose.js

```diff
@@ -1195,6 +1202,8 @@ $(function () {
     });
 });
 
+exports.all_everyone_warn_threshold = all_everyone_warn_threshold;
```

Comment:
You should just declare the export at the top when you declare it, rather than using `var`; that's how we handle this sort of module-level global.